src/Dialogs/DataField.cpp

```diff
@@ -58,11 +60,13 @@ EditDataFieldDialog(const TCHAR *caption, DataField &df,
 
     gdf.ModifyValue(value);
     return true;
-  } else if (df.GetType() == DataField::Type::NUMBER) {
-    unsigned int value = df.GetAsInteger();
```

Comment:
Why remove those lines?

Reply:
The DataFile::Type is INTEGER, not NUMBER  - now a handler is added

Reply:
I don't understand. This doesn't appear to answer my question. If this piece of code is wrong, a separate commit should remove or fix it, and document how it was wrong. I don't see that.

Reply:
... changed to ```} else if (df.GetType() == DataField::Type::INTEGER) {```